ethcore/src/spec/spec.rs

```diff
@@ -67,7 +67,7 @@ pub struct Spec {
 	/// What engine are we using for this?
 	pub engine: Arc<Engine>,
 	/// The fork identifier for this chain. Only needed to distinguish two chains sharing the same genesis.
-	pub fork_name: Option<String>,
+	pub data_dir: String,
```

Comment:
Don't think anyone uses this for private chains. We use it for classic and expanse. Upgrade paths for these is supported.